docs/manual/docs/cmd/graph/schemaextension/schemaextension-set.md

```diff
@@ -0,0 +1,60 @@
+# graph schemaextension add
```

Comment:
This should be `graph schemaextension set` :)

Reply:
How is it possible ? I swear I did not use copy-paste pattern for the new doc file ! 🤣 
Now solved